packages/runtime-version/lib/RuntimeVersion.js

```diff
@@ -18,12 +18,34 @@
 const {OneBehindFetch} = require('amp-toolbox-core');
 const log = require('amp-toolbox-core').log.tag('AMP Runtime Version');
 
-// const CANARY_ENDPOINT = 'https://cdn.ampproject.org/diversions';
-const RELEASE_ENDPOINT = 'https://cdn.ampproject.org/rtv/metadata';
+const RUNTIME_METADATA_ENDPOINT = 'https://cdn.ampproject.org/rtv/metadata';
 
 /**
- * Queries cdn.ampproject.org for the lastest AMP runtime version. Uses a
+ * Queries https://cdn.ampproject.org/rtv/metadata for the lastest AMP runtime version. Uses a
  * stale-while-revalidate caching strategy to avoid refreshing the version.
+ *
+ * More details: https://cdn.ampproject.org/rtv/metadata returns the following metadata:
```

Comment:
I wonder where this information should be in the official AMP docs. We should definitely have this documentation here while we don't get it there.